src/constructs/ses/index.ts

```diff
@@ -0,0 +1,28 @@
+import { Duration } from "aws-cdk-lib";
+import type { EmailIdentityProps } from "aws-cdk-lib/aws-ses";
+import { EmailIdentity, Identity } from "aws-cdk-lib/aws-ses";
+import type { AppIdentity, GuStack } from "../core";
+import { GuCname } from "../dns";
+
+export interface GuEmailIdentityProps extends Omit<EmailIdentityProps, "identity">, AppIdentity {
+  guToolsSubdomain: string;
+}
+
+export class GuEmailIdentity extends EmailIdentity {
+  constructor(scope: GuStack, id: string, props: GuEmailIdentityProps) {
+    super(scope, id, {
+      // Auto verification is only supported for gutools.co.uk domains
```

Comment:
It supports `code.dev-gutools.co.uk` too - I'd suggest we don't hard code the top level domain, since most applications will have the whole domain as a variable they'd like to pass on which they probably wouldn't want to split just to conform to this inner implementation detail

Reply:
I was trying to encode the requirements we want here in the construct, but agreed it is unnecessarily fiddly.

Reply:
perhaps you could have an `endsWith("gutools.co.uk")` (not DOT gutools, to support CODE too) check and throws if other top level domain is used!

Reply:
Ah, neat - I will do this, less complicated than i'd imagined.